README.md

```diff
@@ -81,6 +81,8 @@ Options:
                               source on standard input).
   -S, --skip-string-normalization
                               Don't normalize string quotes or prefixes.
+  -N, --skip-numeric-underscore-normalization
+                              Don't normalize numeric literals with _ separators.			      
```

Comment:
I like the wording in `--help` better (https://github.com/ambv/black/blob/master/black.py#L211): "Don't normalize underscores in numeric literals".

Reply:
I took the one of changelog , sure a thing I 'll can sleep tonight if changed ;-)